task/net/src/main.rs

```diff
@@ -10,15 +10,32 @@ mod buf;
 mod server;
 
 use core::sync::atomic::{AtomicU32, Ordering};
-use stm32h7::stm32h743 as device;
-
+cfg_if::cfg_if! {
```

Comment:
This should probably be controlled by an h743/h753 feature rather than keying off board name. It's possible I just don't understand why you did it this way, however, so, let me know.